client/my-sites/domains/domain-management/edit/style.scss

```diff
@@ -66,23 +66,61 @@
 	.domain-status__card {
 		h2 {
 			font-size: 32px;
+			overflow: hidden;
+			text-overflow: ellipsis;
+
+			@include breakpoint( '<480px' ) {
+				&.mobile-m {
+					font-size: 28px;
+				}
+				&.mobile-l {
+					font-size: 24px;
+				}
+				&.mobile-xl {
+					font-size: 20px;
+				}
+				&.mobile-xxl {
+					font-size: 16px;
+				}
+			}
+
+			@include breakpoint( '>480px' ) {
+				&.desktop-m {
+					font-size: 28px;
+				}
+				&.desktop-l {
+					font-size: 24px;
+				}
+				&.desktop-xl {
+					font-size: 20px;
+				}
+				&.desktop-xxl {
+					font-size: 16px;
+				}
+			}
 		}
 
 		> .domain-status__icon {
 			font-size: 16px;
 			display: flex;
 			align-items: center;
+			border-bottom: 0;
+			padding-bottom: 0;
 			> svg {
 				margin-right: 5px;
 			}
 		}
 
 		> div {
 			margin-bottom: 1.5em;
+			border-bottom: 1px solid #c3c4c7;
```

Comment:
We should be using variables for the colours. In this case, it should be: `var(--color-border-subtle)`